### PR TITLE
Add note to release process for new patch release

### PR DESCRIPTION
**Pull request by lucianvlad**
**What this PR does / why we need it**:
Add note to release process for new patch release to make it clear that we do not need to create a PR and merge it back to main at the end of the new patch release process.
**Which issue this PR fixes** *(use the format `fixes #<issue number>(, fixes #<issue_number>, ...)` to automatically close the issue when PR gets merged)*: fixes #

**Public facing documentation PR** *(if applicable)*

**Special notes for reviewer**:

**Add a screenshot or an example to illustrate the proposed solution:**

**Checklist:**

- [ ] Added relevant notes to [WIP-CHANGELOG.md](https://github.com/elastisys/compliantkubernetes-apps/blob/main/WIP-CHANGELOG.md)
- [ ] Proper commit message prefix on all commits
- [ ] Updated the [public facing documentation](https://github.com/elastisys/compliantkubernetes)
- Is this changeset backwards compatible for existing clusters? Applying:
    - [ ] is completely transparent, will not impact the workload in any way.
    - [ ] requires running a migration script.
    - [ ] will create noticeable cluster degradation.
          E.g. logs or metrics are not being collected or Kubernetes API server
          will not be responding while upgrading.
    - [ ] requires draining and/or replacing nodes.
    - [ ] will change any APIs.
          E.g. removes or changes any CK8S config options or Kubernetes APIs.
    - [ ] will break the cluster.
          I.e. full cluster migration is required.
- Chart checklist (pick exactly one):
    - [ ] I upgraded no Chart.
    - [ ] I upgraded a Chart and determined that no migration steps are needed.
    - [ ] I upgraded a Chart and added [migration steps](https://github.com/elastisys/compliantkubernetes-apps/blob/main/migration).

**Pipeline config** *(if applicable)*
If you change some config options (e.g. add/rename variable or change the default value) you may need to update the config used by the pipeline in `pipeline/config`.

### CODE REVIEW

File: release/README.md [hunk 1]
```diff
@@ -70,3 +70,5 @@ https://semver.org/
     *When the script is done a [GitHub actions workflow pipeline](/.github/workflows/release.yml) should've created a GitHub release from that tag.*
 
 1. Follow the major/minor release from step 6 to update the public release notes.
+
+1. At the end in the github main page you will see a message like `release-X.Y had recent pushes * minutes ago` and the option to `Compare & pull request` --> Ignore this ! Do not create a PR to push back back to main!
```
Comment (robinAwallace) [inline release/README.md:74]:
```suggestion
1. At the end in the github main page you will see a message like `release-X.Y had recent pushes * minutes ago` and the option to `Compare & pull request` --> Ignore this! Do not create a PR to push back back to main!
```

**Review by robinAwallace — APPROVED** [review]

File: release/README.md (hunk 1 above)
Comment (crssnd) [inline release/README.md:74]: more of a personal preference 
```suggestion
> **_WARNING:_** At the end in the github main page you will see a message like `release-X.Y had recent pushes * minutes ago` and the option to `Compare & pull request` --> Ignore this! Do not create a PR to push back to main!
```

**Review by crssnd — APPROVED** [review]

File: release/README.md (hunk 1 above)
Comment (lucianvlad) [reply release/README.md]: done

Comment (lucianvlad) [reply release/README.md]: done

**Review by Xartos — APPROVED** [review]

**Pull request merged**